### PR TITLE
Betting Onboarding

### PR DESCRIPTION
**Pull request by casesandberg**
This PR introduces a different onboarding flow that prompts users to make a few simple bets right away. It then uses the groups from those bets as pre-selected "defaults" for the Interests screen. 

Screenshot 2023-11-08 at 3 34 28 PM

This uses the existing `useABTest` to serve up the experiment until we have access to the more robust Amplitude Experiments.

### To Test:
1. Navigate to `/home?onboarding=intro&forceOnboarding=true` to force show the new onboarding.

### CODE REVIEW

**Review by jahooma — COMMENTED** [review]
Great! Looks good, pretty intuitive and easy for new users.

Some comments after clicking through it:
- On the first step, I don't think it's very obvious that you should click Bet YES or Bet NO. If the bottom bar with the continue button didn't exist, I think that would help a lot
Screenshot 2023-11-21 at 5 48 24 PM
- Clicking the button feels a little slow. Maybe it could show a loading indicator, or immediately show the next question before the bet goes through? Alternately, it could not even place the bets.
- I almost wanted to answer more Yes/No questions than just the 3 provided, like 5?

**Comment by jahooma** [comment]
Has my tentative approval pending making the first step more obvious (to click the bet yes or no).

But also wait for approval from @mantikoros

File: web/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "@amplitude/analytics-browser": "1.9.4",
+    "@amplitude/experiment-js-client": "1.9.5",
```
[the remark this answers is not in the document]
Comment (casesandberg) [reply web/package.json]: Removed.

File: web/lib/service/experiments.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { ENV_CONFIG } from 'common/envs/constants'
```
[the remark this answers is not in the document]
Comment (casesandberg) [reply web/lib/service/experiments.ts]: Removed.

**Comment by casesandberg** [comment]
Thanks @jahooma! 
* Added in name change support to match what Ian just pushed
* Made placing a bet non-blocking so its super quick. 
* Increased the number of trending contracts we fetch to ensure more will be shown. 
* My intuition tells me that hiding the button isn't great UX (consistency is one of the things lacking in the current onboarding) but happy to hide it for now and we can iterate 😄

**Comment by jahooma** [comment]
Hi @casesandberg,

Chatted with Stephen (@mantikoros) who owns the onboarding flow, and he doesn't think this approach is good.

- He thinks the way you answer the questions is misleading and teaches you wrong how betting works, because you don't choose a bet amount.
- It felt tedious to click through, which would slow down people trying to signup. It's easier for them to choose their interests directly through the topic pills. (And it showed too many questions, like 10, which I know we could reduce.)
- Also: He didn't like the copy, it doesn't feature our logo, there's no way to go back to previous screens, and he wanted topic selection to not be skippable.

For these reasons, I think we're not going to merge this PR, even if you were able to fix some of the issues.

In general, we make decisions based off qualitative feedback and refined judgment rather than A/B tests. That helps us move faster.

I realize you put a lot of work into this, so my bad for not getting this signal to you earlier. (Although, I did say that working on onboarding tutorials was not the best idea.)

We're open to other ideas for improving the onboarding of course, if you have other suggestions. Or, I'd be excited if you picked up a different project for manifund. Feel free to message me for further discussion.

**Pull request closed without merging**